### DISCUSSION TITLE
Pushing directly to SafetyGraphics/volcanoPlot

### DISCUSSION THREAD

**Discussion by anhtr** [SafetyGraphics/volcanoPlot#49] [Q&A] [answered]
@jwildfire @bzkrouse 

This is the first time that I work with a repo that belongs to an organization on GitHub. I tried to create a branch and push it to GitHub, but was denied. It seems that I need to get special permission to do push, at least for the software that I use (GitKraken). I wonder how you guys do that.

-----------------------------------
Update: So I found that pushing using command line (Git Bash) worked, but not a third-party GUI. There was an option to ask for permission for GitKraken to access SafetyGraphics organization so the owner of the organization may get an email about that, too.

**Comment by jwildfire** [comment, marked as answer]
Glad you got it working @anhtr. I'll keep an eye out for that notification.
